TricksterBots/Bots/Euchre/EuchreBot.cs

```diff
@@ -105,6 +113,125 @@ public override BidBase SuggestBid(SuggestBidState<EuchreOptions> state)
             return canBidSuggestion ? suggestion : legalBids.FirstOrDefault(b => b.value == BidBase.Pass) ?? legalBids.First();
         }
 
+        private BidBase SuggestBidEuchreBid(SuggestBidState<EuchreOptions> state)
+        {
+            var legalBids = state.legalBids.Where(b => b.value != BidBase.NoBid).Select(b => new BidEuchreBid(b.value)).ToList();
+            var legalLevelBids = legalBids.Where(b => b.IsLevelBid).ToList();
+            var maxTricks = 0.0;
+            var maxSuit = Suit.Unknown;
+            var ntDown = false;
+            var players = new PlayersCollectionBase(this, state.players);
+            var partners = players.PartnersOf(state.player);
+            var possibleTricks = (double)state.hand.Count;
+            var willLeadFirst = (state.dealerSeat + 1) % state.options.players == state.player.Seat;
```

Comment:
Also need to account for "alone leads left" later on before bidding alone of any kind in NT.